### ISSUE TITLE
Overlay beneath focused window

### ISSUE THREAD

**Issue by PavanLuca** [languagetool-org/languagetool#10690]
The overlay was being displayed beneath the focused window so I couldn't use it.
![Schermata del 2024-06-26 13-23-59](https://github.com/languagetool-org/languagetool/assets/123581658/e65c8976-9769-4055-892e-3e9e5c5b9d3f)

**Comment by danielnaber** [comment]
How exactly do you get to that view? Using "modify post" at Discord, the UI looks different for me and I can't reproduce the issue.

**Comment by PavanLuca** [comment]
It's different because I apply dark themes to pages in light theme with Dark Reader. That extension doesn't cause the issue though, disabling it the overlay is still displayed beneath the window. Looks that LanguageTool doesn't detect that the focus is on the new window and still loads the overlay beneath the window. Should I post all logs of the console after loading LanguageTool beneath that window a bit of times to see something useful?

**Comment by danielnaber** [comment]
To fix this, we'll need detailed instructions on how to reproduce it. On which URL does it happen, which buttons need to be clicked to make this happen?

**Comment by PavanLuca** [comment]
Alright, I thought you understood it when you wrote “Modify Post”, it's exactly there, on https://support.discord.com/hc/it/community but I see it because I can edit one of my posts, so in my personal profile page. I tried pointing the overlay but fails, other nodes were being correctly highlighted instead.
![immagine](https://github.com/languagetool-org/languagetool/assets/123581658/719a6647-7d1c-44d9-8430-7eb023db2a98)

**Comment by danielnaber** [comment]
Maybe my problem was that I tried with a chat post, your post seems to be of a different type?

**Comment by PavanLuca** [comment]
Yes, you won't see it by creating a post because it involves EDITING it. Clicking my avatar on top right corner I can go to **My profile**, click a post of mine, edit it and see the overlay beneath the editing window. If you have an account there you can do the same to see if you can reproduce the issue, you can then delete the post after testing.
